website/docs/r/lb_service_monitor.html.markdown

```diff
@@ -0,0 +1,95 @@
+
+## Example Usage
+
+```hcl
+provider "vcd" {
+  user     = "${var.admin_user}"
+  password = "${var.admin_password}"
+  org      = "System"
+  url      = "https://AcmeVcd/api"
+}
+
+resource "vcd_lb_service_monitor" "monitor" {
+  org          = "my-org"
+  vdc          = "my-org-vdc"
+  edge_gateway = "my-edge-gw"
+
+  name        = "testing-monitor-yy"
+  interval    = "5"
+  timeout     = "20"
+  max_retries = "3"
+  type        = "http"
+  http_method = "GET"
+  url         = "/health"
+  extension = {
+    content-type = "application/json"
+    linespan     = ""
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `edge_gateway` - (Required) The name of the edge gateway on which the service monitor is to be created
+* `vdc` - (Optional) The name of VDC to use, optional if defined at provider level
+* `org` - (Optional) The name of organization to use, optional if defined at provider level. Useful when connected as sysadmin working across different organisations
+* `name` - (Required) Service Monitor name
+
+* `interval` - (Required) Interval at which a server is to be monitored using the specified Method. 
+* `timeout` - (Required) Maximum time in seconds within which a response from the server must be received 
+* `max_retries` - (Required) Number of times the specified monitoring Method must fail sequentially before the server is declared down
+* `type` - (Required) Select the way in which you want to send the health check request to the server — `http`, `https`, 
+`tcp`, `icmp`, or `udp`. Depending on the type selected, the remaining attributes are allowed or not
+* `method` - (Optional) For types `http` and `https`. Select http method to be used to detect server status. One of OPTIONS, GET, HEAD, POST, PUT, DELETE, TRACE, or CONNECT
+* `url` - (Optional) For types `http` and `https`. URL to be used in the server status request
+* `send` - (Optional) For types `http`,  `https`, and `udp`. The data to be sent.
+* `expected` - (Optional) For types `http` and `https`. String that the monitor expects to match in the status line of 
+the HTTP or HTTPS response (for example, `HTTP/1.1`)
+* `receive` - (Optional) For types `http`,  `https`, and `udp`. The string to be matched in the response content.
+**Note**: When `expected` is not matched, the monitor does not try to match the Receive content
+* `extension` - (Required) A map of advanced monitor parameters as key=value pairs (i.e. `max-age=SECONDS`, `invert-regex`)
+**Note**: When you need a value of `key` only format just set value to empty string (i.e. `linespan = ""`)
+
+## Attribute Reference
+
+The following attributes are exported on the base level of this resource:
+
+* `id` - The ID of the load balancer service monitor
+
+## Importing
+
+An existing load balancer service monitor can be [imported][docs-import] into this resource
+via supplying the full dot separated path for load balancer service monitor. An example is below:
```

Comment:
I'm thinking about a warning here that after import you need to do manual operations. As it stands right now, the user may be tempted to run the import command and think that all is done.

Reply:
Done.